src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/MinorAlleleFractionCacheKey.java

```diff
@@ -0,0 +1,43 @@
+package org.broadinstitute.hellbender.tools.exome.allelefraction;
```

Comment:
I would nest this in `MinorAlleleFractionCache`, but don't mind it like as a separate class either.